src/transforms/postcss.js

```diff
@@ -51,7 +51,15 @@ async function getConfig(asset) {
 
   if (asset.options.minify) {
     config.plugins.push(
-      cssnano((await asset.getConfig(['cssnano.config.js'])) || {})
+      cssnano(
+        (await asset.getConfig(['cssnano.config.js'])) || {
```

Comment:
Maybe change this to Object.assign?
So config doesn't overwrite defaults but appends them

Reply:
That's reasonable, but how would we communicate the net result to users. I think it's easier to reason about if we just use the `cssnano.config.js` as provided (_advanced user, let them choose_), and otherwise, just use safe defaults.

Reply:
Alright, fair enough